alamo_alerter/plugins/image_renderer.py

```
@@ -0,0 +1,51 @@
+import logging
+
+from alamo_alerter.conf import settings
+import requests
+
+
+logger = logging.getLogger(__name__)
+
+
+class ImageRendererPlugin(object):
+    def __init__(self):
+        logger.debug('Image renderer was loaded')
+
+    def __call__(self, check, trigger, *args, ** kwargs):
+        for source in self._get_trigger_sources(check, trigger):
+            if not self._can_render_image(source):
+                continue
+            url = self._submit_job(source)
```

shouldn't we catch the errors and log them instead of failing hard ?